webapp/src/utils/jira_issue_metadata.tsx

```diff
@@ -25,6 +26,8 @@ type FieldWithInfo = JiraField & {
     issueTypeMeta: IssueTypeIdentifier;
 }
 
+export const FIELD_KEY_STATUS = 'status';
```

Comment:
are we using this constant outside this file ? Mainly importing it somewhere else

Reply:
Yes